README.md

```diff
@@ -14,6 +14,10 @@ _Do not specify default parameters if you don't want to override them_ (just bec
 
 # Changes
 
+## 3.4.2
++ optimized converter Value.SIMPLE_VALUE to Java Class
++ removed method call MessageWrapper.cloneMessage to improve performance
```

Comment:
Please write all release notes from th2-codec-sailfish between old and new releases